apps-rendering/src/components/series.tsx

```diff
@@ -63,10 +63,20 @@ const immersiveLabsLinkStyles = css`
 
 const blogLinkStyles = (format: ArticleFormat): SerializedStyles => css`
 	${headline.xxxsmall({ lineHeight: 'tight', fontWeight: 'bold' })}
-	color: ${format.design === ArticleDesign.LiveBlog
+	color: ${format.design === ArticleDesign.LiveBlog ||
```

Comment:
I could be wrong, but I think this function is only ever called in the context of a LiveBlog or a DeadBlog so this condition will always be true?

Reply:
I think you're right!